### DISCUSSION TITLE
Got removed or accidentally the Discord within minutes after joining, can't rejoin

### DISCUSSION THREAD

**Discussion by NoahNim** [Pycord-Development/pycord#2169] [Q&A] [answered]
Hello! I joined the Discord server a few hours ago to have a look around and search the help section. I never posted or said anything.   I am unable to rejoin using the publicly available Discord link.

**Comment by JustaSqu1d** [comment, marked as answer]
If you could provide your Discord username (e.g. @foo.bar or @foo.bar#1234), I can look into why you were removed from the server (if at all). 

Please note that this discussion forum is only for library development questions and not general questions or support.

**Comment by Lulalaby** [comment]
@NoahNim please send me your user id via lala@pycord.dev
